modules/sensitive/sensitive.go

```diff
@@ -0,0 +1,32 @@
+package sensitivemod
+
+import (
+	"github.com/king133134/sensfilter"
+	"log"
+	"os"
+	"strings"
+)
+
+type Checker struct {
+	checker *sensfilter.Search
+}
+
+func NewChecker() *Checker {
+	checkerType := os.Getenv("SENSITIVE_TYPE")
+	log.Println("Checker Type:" + checkerType)
+	var search *sensfilter.Search
+	switch checkerType {
+	case "string":
+		sensitiveWord := os.Getenv("SENSITIVE_WORD")
+		wordArray := strings.Split(sensitiveWord, ",")
+		search = sensfilter.Strings(wordArray)
+	default:
+		log.Println("Bad checker type")
+		return nil
```

Comment:
if the newXXX function called, an error should be returned to prevent potential module calls from failing. the fx will check options return, and ensure no error before the program real started.

Reply:
add new checker error